integrations/bitdefender/nango.yaml

```diff
@@ -0,0 +1,111 @@
+integrations:
+    bitdefender:
+        syncs:
+            companies:
+                version: 1.0.0
+                runs: every hour
+                description: |
+                    Syncs company details from Bitdefender GravityZone. 
+                    This provides basic information about the company such as name, 
+                    type, timezone, and subscribed services.
+                output: BitdefenderCompany
+                sync_type: full
+                track_deletes: true
+                endpoint: GET /v1.0/jsonrpc/companies/details
```

Comment:
The endpoint shouldn't reflect what is called in the script but rather should represent how a user would call the endpoint to initiate within nango. So this endpoint can just be `/companies`